ru/lessons/libraries/bypass.md

```diff
@@ -0,0 +1,272 @@
+---
+version: 1.0.0
+title: Bypass
+---
+
+При тестировании приложений часто идет речь о запросах к внешним сервисам.
+Мы даже можем захотеть симулировать различные ситуации типа неожиданных ошибок со стороны сервера.
+Реализация подобного подхода в Эликсире требует дополнительных инструментов.
```

Comment:
```suggestion
Реализация подобного подхода в Elixir требует дополнительных инструментов.
```
IIRC we [didn't translate the language name](https://github.com/elixirschool/elixirschool/search?q=%D0%AD%D0%BB%D0%B8%D0%BA%D1%81%D0%B8%D1%80).